src/index.js

```diff
@@ -2,26 +2,16 @@
  * @flow
  */
 
-// example of generating HTML with js
-function renderSidebar() {
-  const sidebarContents = `
-    <h2 class="email-header">Inbox</h2>
-    <ul class="email-list">
-      <li>
-        <button class="email-item" type="button">
-          <div class="sender-details">
-            <p>Email sender</p>
-            <span>Timestamp</span>
-          </div>
-          <p class="email-subject">Email1 subject</p>
-          <p>Email snippet</p>
-        </button>
-      </li>
-    </ul>
-  `;
+import store from './store/index';
+import * as t from './ui/templates';
 
-  const container = document.querySelector('.email-list-container');
-  if (container != null) container.innerHTML = sidebarContents;
+function render(selector, template) {
```

Comment:
👍

Reply:
Awesome to split to different modules! Even though right now it's a little hard for me understand all of them, hhhhh.